generator/maxspeed_collector.cpp

```diff
@@ -0,0 +1,66 @@
+#include "generator/maxspeed_collector.hpp"
+
+#include "coding/file_writer.hpp"
+
+#include "base/geo_object_id.hpp"
+#include "base/logging.hpp"
+
+#include <fstream>
+#include <sstream>
+
+namespace feature
+{
+using namespace base;
+using namespace std;
+
+void MaxspeedCollector::Process(OsmElement const & p)
```

Comment:
nit: s/p/el/ (or s/el/p/ in the hpp file)

Reply:
Fixed.